### ISSUE TITLE
About the learning rate in scheduler

### ISSUE THREAD

**Issue by Arios42** [Tony-Y/pytorch_warmup#15]
I follow the tutorial to implement the warmup_scheduler, but the learning rate I get from the get_lasr_lr() of the torch.optim.lr_scheduler.MultiStepLR is the same as the initial learning rate. How should I get the learning rate after the warmup process?

**Comment by Tony-Y** [comment]
The learning rate can be got as the following:

https://github.com/Tony-Y/pytorch_warmup/blob/ed2b7bdfa43cd11b346fd4bfb2f5f7f055531bab/examples/emnist/main.py#L35
